[2024-10-09T05:16:51.021+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: Daily_Gold_Report_Generate.daily_report_generate scheduled__2024-10-05T00:00:00+00:00 [queued]>
[2024-10-09T05:16:51.034+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: Daily_Gold_Report_Generate.daily_report_generate scheduled__2024-10-05T00:00:00+00:00 [queued]>
[2024-10-09T05:16:51.035+0000] {taskinstance.py:2193} INFO - Starting attempt 2 of 4
[2024-10-09T05:16:51.064+0000] {taskinstance.py:2217} INFO - Executing <Task(PythonOperator): daily_report_generate> on 2024-10-05 00:00:00+00:00
[2024-10-09T05:16:51.102+0000] {standard_task_runner.py:60} INFO - Started process 790 to run task
[2024-10-09T05:16:51.118+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'Daily_Gold_Report_Generate', 'daily_report_generate', 'scheduled__2024-10-05T00:00:00+00:00', '--job-id', '301', '--raw', '--subdir', 'DAGS_FOLDER/gold_daily_report_generate_pipeline.py', '--cfg-path', '/tmp/tmp7emwqgop']
[2024-10-09T05:16:51.119+0000] {standard_task_runner.py:88} INFO - Job 301: Subtask daily_report_generate
[2024-10-09T05:16:51.347+0000] {task_command.py:423} INFO - Running <TaskInstance: Daily_Gold_Report_Generate.daily_report_generate scheduled__2024-10-05T00:00:00+00:00 [running]> on host 68dba313a760
[2024-10-09T05:16:54.868+0000] {taskinstance.py:2513} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='sdeshan' AIRFLOW_CTX_DAG_ID='Daily_Gold_Report_Generate' AIRFLOW_CTX_TASK_ID='daily_report_generate' AIRFLOW_CTX_EXECUTION_DATE='2024-10-05T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-10-05T00:00:00+00:00'
[2024-10-09T05:17:01.134+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.8/site-packages/sklearn/base.py:465 UserWarning: X does not have valid feature names, but StandardScaler was fitted with feature names
[2024-10-09T05:27:00.995+0000] {local_task_job_runner.py:121} ERROR - Received SIGTERM. Terminating subprocesses
[2024-10-09T05:27:01.110+0000] {process_utils.py:131} INFO - Sending 15 to group 790. PIDs of all processes in the group: [790]
[2024-10-09T05:27:01.124+0000] {process_utils.py:86} INFO - Sending the signal 15 to group 790
[2024-10-09T05:27:03.235+0000] {local_task_job_runner.py:121} ERROR - Received SIGTERM. Terminating subprocesses
[2024-10-09T05:27:03.239+0000] {process_utils.py:131} INFO - Sending 15 to group 790. PIDs of all processes in the group: [790]
